src/OrleansSQLUtils/Storage/Provider/AdoNetStorageProvider.cs

```diff
@@ -200,10 +202,12 @@ public class AdoNetStorageProvider: IStorageProvider
                 queries.Single(i => i.Item1 == "ReadFromStorageKey").Item2,
                 queries.Single(i => i.Item1 == "ClearStorageKey").Item2);
 
-            Log = providerRuntime.GetLogger(GetType().FullName);
+            var loggerFactory = providerRuntime.ServiceProvider.GetService<ILoggerFactory>();
+            this.logger = loggerFactory.CreateLogger<AdoNetStorageProvider>();
+            Log = new LoggerWrapper(this.logger, this.GetType().FullName, loggerFactory);
```

Comment:
Do we need the wrapper here?

You should be able to get `ILogger<AdoNetStorageProvider>` from the container directly, with no factory.

Reply:
that is for IStorageProvider.Logger method. There's discussion about moving that to a legacy package, or even remove it. But I don't want to involve that in this PR yet.

Reply:
This is the same with what I did in #3611